README.md

```diff
@@ -387,6 +391,79 @@ $ make flash
 [here](https://github.com/96boards/documentation/wiki/HiKeyUEFI#flash-binaries-to-emmc-))
 
 The board is ready to be booted.
+#### 5.5.2 Debian based / 96boards RPB (experimental)
+Start by getting the source and toolchain (see above), then continue by
+downloading the system image (root fs). There are two versions, the one that
+works best has a click through license which means that you have to manually
```

Comment:
* what is working "wrong" with the other?
* where to find the rootfs image to download?